h/static/scripts/annotation-mapper.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+
+// Fetch the container object for the passed annotation from the threading
```

Comment:
What is a 'container' for an annotation and what's the structure of annotations in the threading service (if the answer to this is already in the docs somewhere that I missed, feel free to just point me at it)?